app/models/fixed_asset.rb

```diff
@@ -193,6 +193,10 @@ class FixedAsset < Ekylibre::Record::Base
     depreciate! if @auto_depreciate
   end
 
+  def on_unclosed_periods?
+    self.started_on <= self.journal.closed_on ? false : true
```

Comment:
The ternary operator feels unnecessary here. Changing the operator (`>` is logically symmetrical to `<=`) would allow us to remove it, making for a cleaner method body.